diff --git a/src/handler/MapDrag.js b/src/handler/MapDrag.js
index e027ab2..d3681b5 100644
--- a/src/handler/MapDrag.js
+++ b/src/handler/MapDrag.js
@@ -61,7 +61,7 @@ L.Handler.MapDrag = L.Handler.extend({
 			newX2 = (x + halfWidth + dx) % worldWidth - halfWidth - dx,
 			newX = Math.abs(newX1 + dx) < Math.abs(newX2 + dx) ? newX1 : newX2;
 
-		this._draggable._newPos.x = newX;
+		//this._draggable._newPos.x = newX;
 	},
 
 	_onDragEnd: function() {
